### PR TITLE
data

### PR DESCRIPTION
**Pull request by namlxag98**

### CODE REVIEW

**Comment by namlxag98** [comment]
hi

File: server/.migrate [hunk 1]
```diff
@@ -3,11 +3,11 @@
   "migrations": [
```
Comment (vtquy98) [inline server/.migrate]: add .migrtae to .gitignore or ``don't commit`` this file

File: server/server.js [hunk 2]
```diff
@@ -8,11 +8,13 @@ import logger from "morgan";
...
 // import session from "./middlewares/session";
 import session from "express-session";
+// import { userRoute, auctionCreateRouter } from "./routes/index";
+import auctionCreateRouter from "./routes/auctionCreationRouter";
 
 const app = express();
 app.disable("x-powered-by");
 app.use(helmet());
-app.use(cors());
+app.use(cors()); // 2 file client, server
```
Comment (vtquy98) [inline server/server.js]: what for ?

File: server/web3/auctionCreation.js [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
+import auctionCreation from "../contract-build/AuctionFactory.json";
 // import { Contracts } from '../../services';
 
 //how to get contract address on db!!!??
 //this is unique deployed contract!
 
 const instance = new web3.eth.Contract(
-    auctionCreation.abi,
+  auctionCreation.abi,
   process.env.CONTRACT_ADDRESS
```
Comment (vtquy98) [inline server/web3/auctionCreation.js]: review ``CONTRACT_ADDRESS`` in .env

**Pull request merged**